pkg/handler/v1/provider/openstack.go

```diff
@@ -28,34 +28,44 @@ import (
 	"k8c.io/kubermatic/v2/pkg/provider"
 )
 
+type credential struct {
+	credential, username, password, tenant, tenantID, domain string
```

Comment:
`credential` does not seem to be necessary

Reply:
used here: https://github.com/kubermatic/kubermatic/pull/6852/files#diff-1ea1ef7250db578e0ba92bef24d40f20707fb444fb51c91f5fa05b14745a6bc5R325

Reply:
I see check my comment https://github.com/kubermatic/kubermatic/pull/6852/files#r611916775

Reply:
ok so I can remove `credential` field form struct and just do `presetName= req.Credential`

Reply:
yep